app/src/main/java/com/framgia/music_44/screen/home/HomeFragment.java

```diff
@@ -22,17 +22,28 @@
     private HomeAdapter mHomeAdapter;
     private List<Songs> mSongs;
     private Navigator mNavigator;
+    private HomeContract.Presenter presenter;
+    public static final String ARGUMENT_GENRES = "ARGUMENT_GENRES";
 
     public static HomeFragment newInstance() {
         return new HomeFragment();
     }
 
+    public static HomeFragment getListMusicFragment(String genres) {
```

Comment:
rename method and following conventation
```suggestion
     public static HomeFragment newInstance(String genres) {
```